Fix documentation issues

capelastegui opened the pull request:
close #72 #66

slenas commented on anticipy/forecast.py:
```diff
@@ -655,7 +656,7 @@ def extrapolate_model(
         (untested) Any date unit or time unit accepted by numpy should also
         work, see
-        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits # noqa
+        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits #noqa
```
Travis complains about this line. Probably it needs the space.

slenas commented on anticipy/forecast_models.py:
```diff
@@ -250,35 +251,51 @@ class ForecastModel:
+        # Get model output
+        y = f_model(a_x, a_date, parameters)
 
     The following pre-generated models are available. They are available as attributes from this module:
```
This line also needs fixing. Travis complains about being too long.

slenas commented on anticipy/forecast_plot.py:
```diff
@@ -329,12 +329,12 @@ def plot_forecast(df_fcast, output, path=None, width=None, height=None,
+      |  - model (str) : ID for the forecast model
+      |  - y (float) : Value of the time series in that sample
+      |  - is_actuals (bool) : True for actuals samples, False for forecast
+      |
```
what's the purpose of this blank line?

capelastegui replied:
Bug - I'll remove it.

slenas commented on anticipy/model_utils.py:
```diff
@@ -128,7 +128,7 @@ def get_s_x_extrapolate(
         samples, or 'D' for daily samples. (untested) Any date unit or time
         unit accepted by numpy should also work, see
-        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits  # noqa
+        https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.datetime.html#arrays-dtypes-dateunits #noqa
```
needs the space apparently

slenas commented on docs/source/tutorial.rst:
```diff
@@ -81,7 +81,7 @@ An input dataframe should meet the following constraints:
   - y column may include null values.
-  - Multiple values per sample may be included, e.g when the same metric is observed by multiple sources. In that case,
+  - Multiple values per sample may be included, e.g. when the same metric is observed by multiple sources. In that case,
     it is possible to assign weights to each individual sample so that one source will be given higher priority. To do
```
Does it get to the next line here?

slenas replied:
if it's a continuing line no problem then. It seemed that there were more spaces there so the line was breaking without an actual reason...

capelastegui commented:
I have fixed the problem with 3 long lines that made pep8 complain. I had started by removing all # noqa annotations, but these lines had hyperlinks that couldn't be split, so I added back the noqa - with a typo. This is again working as intended. 

BTW, there is a known problem with # noqa annotations in docs and sphinx: I haven't found any way to make sphinx aware of pep8 annotations, so the text 'noqa' is actually showing up in our docs. The best solution is to fix the code problems so that the annotations are no longer necessary. Unfortunately, we can't really do that with hyperlinks that are more than 80 characters long. Any suggestions are welcome, but for now I think we'll merge with the slightly bugged docs.

capelastegui commented:
One more thing... after committing fixes, I saw that the comment UI in github has an option to propose quick code changes from the web. We could try that out the next time we have a bunch of single-character errors.
